cli/src/main/java/com/devonfw/tools/ide/tool/LocalToolCommandlet.java

```diff
@@ -257,8 +258,9 @@ public boolean installAsDependency(VersionRange version, ProcessContext processC
                 + " and this tool does not support the software repository.");
       }
       this.context.info(
-          "Configured version of tool {} is {} but does not match version to install {} - need to use different version from software repository.",
-          this.tool, configuredVersion, version);
+          "The tool {} requires {} in version range {} but your project is using {} version {} that does not match."
+              + " Hence, we install an additional {} version {} to run intellij.",
```

Comment:
At the end you still have `intellij` hardcoded.
Maybe you can also rephrase the message to avoid repeating yourself so much often and to avoid providing the same value multiple times as log argument.